packages/lesswrong/components/users/UsersNameDisplay.tsx

```diff
@@ -58,16 +75,35 @@ const UsersNameDisplay = ({user, color=false, nofollow=false, simple=false, clas
     </span>
   }
 
-  return <span {...eventHandlers} className={className}>
-    <AnalyticsContext pageElementContext="userNameDisplay" userIdDisplayed={user._id}>
-    <LWTooltip title={<UserTooltip user={user}/>} placement={tooltipPlacement} inlineBlock={false}>
-      <Link to={userGetProfileUrl(user)} className={colorClass}
-        {...(nofollow ? {rel:"nofollow"} : {})}
+  const showNewUserIcon = allowNewUserIcon && isNewUser(user);
+  return <span className={className}>
+    <span {...eventHandlers}>
+      <AnalyticsContext pageElementContext="userNameDisplay" userIdDisplayed={user._id}>
+      <LWTooltip title={<UserTooltip user={user}/>} placement={tooltipPlacement} inlineBlock={false}>
+        <Link to={userGetProfileUrl(user)} className={colorClass}
+          {...(nofollow ? {rel:"nofollow"} : {})}
+        >
+          {displayName}
+        </Link>
+      </LWTooltip>
+      </AnalyticsContext>
+    </span>
+    {showAuthorIcon && <LWTooltip
+        placement="bottom-start"
+        title="Post author"
+        className={classes.iconWrapper}
+      >
+        <ForumIcon icon="Author" className={classes.postAuthorIcon} />
+      </LWTooltip>
+    }
+    {showNewUserIcon && <LWTooltip
+        placement="bottom-start"
+        title={`${user.displayName} is either new on ${siteNameWithArticleSetting.get()} or doesn't have much karma yet.`}
+        className={classes.iconWrapper}
       >
-        {displayName}
-      </Link>
-    </LWTooltip>
-    </AnalyticsContext>
+        <ForumIcon icon="Sprout" className={classes.sproutIcon} />
```

Comment:
Woah, I didn't notice us add this. This is so good. Who get's the praise for this? @oetherington, nice work!